pom.xml

```diff
@@ -431,14 +443,14 @@
     <version.org.apache.activemq>1.5.5.jbossorg-008</version.org.apache.activemq>
     <version.org.hibernate.javax.persistence>1.0.1.Final</version.org.hibernate.javax.persistence>
     <version.org.jboss.spec.javax.interceptor>1.0.0.Final</version.org.jboss.spec.javax.interceptor>
-    <version.org.jboss.spec.javax.transaction>1.0.0.Final</version.org.jboss.spec.javax.transaction>
+    <version.org.jboss.spec.javax.transaction>1.1.0.Final</version.org.jboss.spec.javax.transaction>
     <version.org.jboss.spec.javax.ejb>1.0.2.Final</version.org.jboss.spec.javax.ejb>
     <version.org.jboss.spec.javax.servlet>1.0.2.Final</version.org.jboss.spec.javax.servlet>
     <version.org.jboss.spec.jboss-javaee-6.0>3.0.2.Final</version.org.jboss.spec.jboss-javaee-6.0>
     <version.org.jacorb.jacorb>2.3.2-jbossorg-5</version.org.jacorb.jacorb>
     <version.org.jacorb.jacorb-idl-compiler>2.3.1</version.org.jacorb.jacorb-idl-compiler>
     <version.jacorb>2.3.1jboss.patch01-brew</version.jacorb>
-    <version.org.jboss.openjdk-orb>8.0.4.Final</version.org.jboss.openjdk-orb>
+    <version.org.jboss.openjdk-orb>8.0.8.Final</version.org.jboss.openjdk-orb>
```

Comment:
Is this the same version as is used by WFLY. Could we pull in the version from jboss-parent?

Reply:
No, it does not include in the jboss-parent.